src/components/20-molecules/file-upload/ui.test.js

```diff
@@ -252,11 +252,10 @@ fixture('File upload - Remove file event').page(
   `${host}/iframe.html?id=examples-file-upload-react--story&viewMode=story`
```

Comment:
We once decided that we won't touch testcafe tests anymore. Rather we migrate to playwright, if any test is failing. :)